nomenklatura/cli.py

```diff
@@ -139,7 +141,8 @@ def dedupe(path: Path, xref: bool = False, resolver: Optional[Path] = None) -> N
     resolver_ = _get_resolver(path, resolver)
     store = load_entity_file_store(path, resolver=resolver_)
     if xref:
-        run_xref(resolver_, store)
+        index_dir = path.parent / INDEX_SEGMENT
```

Comment:
There's a world where we may want this to check an env var for a NOMENKLATURA_INDEX_PATH before cluttering the working directory?

Reply:
will do. Should I add a note to the README too?

Should I also update the readme to say install `nomenklatura[tantivy]` since it defaults to that index.